aggregator_api/src/routes.rs

```diff
@@ -281,15 +278,15 @@ pub(super) async fn get_task<C: Clock>(
 pub(super) async fn delete_task<C: Clock>(
     conn: &mut Conn,
     State(ds): State<Arc<Datastore<C>>>,
-) -> Result<impl Handler, Status> {
+) -> Result<Status, Status> {
     let task_id = conn.task_id_param()?;
 
     ds.run_tx_with_name("delete_task", |tx| {
         Box::pin(async move { tx.delete_task(&task_id).await })
     })
     .await
     .map_err(|err| match err {
-        datastore::Error::MutationTargetNotFound => Status::NotFound,
+        datastore::Error::MutationTargetNotFound => Status::NoContent,
```

Comment:
Were there not tests that had to be updated to account for this? If not, we should take the opportunity to add tests that exercise this.

Reply:
Whoops, that got lost in a `git stash`.